### DISCUSSION TITLE
Proforma

### DISCUSSION THREAD

**Discussion by aadmant** [NREL/REopt-API-Analysis#82] [1 - General REopt Questions] [answered]
Hello, 

I have a question, I do not know if it is only the case with me, but the Profoma spreedsheet can not be downloaded! I get the message "We are sorry but something went wrong". Can someone help me with this problem.

With kind regards

**Comment by LParkhill** [comment]
Thanks for bringing this bug to our attention! It appears to be impacting every evaluation. The developers have been notified and they will get it fixed as soon as possible.

**Comment by LParkhill** [comment, marked as answer]
@aadmant thanks for your patience. The pro forma link was more complex to fix than expected. However, it was fixed yesterday and should be working for you now.
